### DISCUSSION TITLE
这个错误需要改哪里？ERROR: for redis  UnixHTTPConnectionPool(host='localhost', port=None): Read timed out.

### DISCUSSION THREAD

**Discussion by xiaodesetingyongzhanghao** [qd-today/qd#372] [Q&A]
ERROR: for redis  UnixHTTPConnectionPool(host='localhost', port=None): Read timed out. (read timeout=60)

ERROR: for redis  UnixHTTPConnectionPool(host='localhost', port=None): Read timed out. (read timeout=60)
ERROR: An HTTP request took too long to complete. Retry with --verbose to obtain debug information.
If you encounter this issue regularly because of slow network conditions, consider setting COMPOSE_HTTP_TIMEOUT to a higher value (current value: 60).

**Comment by a76yyyy** [comment]
能连上redis?

**Reply by xiaodesetingyongzhanghao** [reply]
> 能连上redis？
> 

(；′⌒`)纯小白不太懂，就按着Docker 容器部署 教程来得
是不是和我的Docker版本不兼容？

**Reply by a76yyyy** [reply]
和Redis断连或者没连上Redis，正常情况下重启Redis或者重启容器可以解决问题

如果Redis确认已经开启了的话，请检查Redis和QD是不是在一个容器网络下

**Reply by a76yyyy** [reply]
如果还报错请reopen这个discussion

**Reply by xiaodesetingyongzhanghao** [reply]
> 如果还报错请reopen这个discussion

好的  我看看  感觉是硬件比较差 用矿渣弄的
